book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author
```

Comment:
- [ ] As raised earlier, kindly add @rentals to be part of the attribute accessor. 👍 
```suggestion
class Book
  attr_accessor :title, :author, :rentals
```